datacommons/statvar.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data Commons Python API Stat Module.
+
+Provides functions for getting data on StatVars from Data Commons Graph.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from datacommons.utils import _API_ROOT, _API_ENDPOINTS, _ENV_VAR_API_KEY
+
+import json
+import os
+import six.moves.urllib.error
+import six.moves.urllib.request
+
+import datacommons.utils as utils
+
+
+def _send_get_stat_req(url):
```

Comment:
this seems mostly identical with _send_request -- does that not work for this module?

Reply:
_send_request is expecting a payload keyword, but the REST API does not have that for /stat. It's an open item for discussion.

Reply:
i wouldn't fork just for that reason. can we add support for non-payload requests?